### DISCUSSION TITLE
Reset ArchWSL to it's initial state

### DISCUSSION THREAD

**Discussion by ghost** [yuk7/ArchWSL#324] [Q&A]
How would I go about resetting ArchWSL to it's initial immediately post-install state ? 
I'm experimenting around with it a lot with the end goal of just having a bunch of script which I can replicate my development setup on any future ArchWSL.

**Comment by jfdtse** [comment]
I think it's easier to install a second instance. Just unzip rootfs.tar.gz and import it:

`wsl --import Arch2 C:\WSL\Arch2 C:\Users\myuser\Downloads\rootfs.tar.gz`

**Reply by ghost** [reply]
And then unregister the first one ? I'm using scoop. While I can adapt the command, both should be in the persist folder can't I just overwrite rootfs ?

**Reply by jfdtse** [reply]
You can unregister the first or have both in separated folders.

Have several WSL instances is as easier as import rootfs in another folder.

Check `wsl --help` to see the entire command help.
